### DISCUSSION TITLE
Vehicle sample project (and Vehicle component?)

### DISCUSSION THREAD

**Discussion by NicusorN5** [stride3d/stride#2113] [Ideas]
Hi, everyone, I've recently was thinking about a vehicle simulator type of sample project for some time. 

I've seen that vehicle mechanics were discussed in https://github.com/stride3d/stride/discussions/1800, but I'm talking about a separate project.

I could begin by implementing a "Vehicle" component (https://github.com/stride3d/stride/issues/138) , or by using separate scripts?

Except that I'm not sure how to mess with the physics APIs, what do you think about this idea?

**Comment by Eideren** [comment]
@Nicogo1705 has a simple vehicle implementation in one of his demos for bepu https://github.com/Nicogo1705/Stride.BepuPhysics
The project should be integrated soon enough, something to keep an eye on.
